homeassistant/components/bluesound/media_player.py

```diff
@@ -168,7 +168,7 @@ async def async_will_remove_from_hass(self) -> None:
     @callback
     def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
-        self._attr_available = self.coordinator.data.is_online
+        self._attr_available = self.coordinator.last_update_success
```

Comment:
You don't need to set `_attr_available` anymore. If you inherit CoordinatorEntity, the default implementation already returns `self.coordinator.last_update_success`